cpp/log/etcd_consistent_store-inl.h

```diff
@@ -314,6 +386,73 @@ std::string EtcdConsistentStore<Logged>::GetFullPath(
   return root_ + key;
 }
 
+
+// static
+template <class Logged>
+template <class T>
+Update<T> EtcdConsistentStore<Logged>::TypedUpdateFromWatchUpdate(
```

Comment:
Ooh, I see... Add a comment to explain that, so people (which might be "future me") don't come asking you about this again. :wink: